server/eclipse-project/src/main/java/us/freeandfair/corla/csv/DominionCVRExportParser.java

```diff
@@ -223,6 +235,20 @@ private void addContests(final CSVRecord the_choice_line,
   }
   
   /**
+   * Checks to see if the set of parsed CVRs needs flushing, and does so 
+   * if necessary.
+   */
+  private void checkForFlush() {
+    if (my_parsed_cvrs.size() % BATCH_SIZE == 0) {
```

Comment:
I don't think it's worth doing at the moment, since it occurs only in two unrelated parsers that don't really need to have the same batch size, but maybe in the future...